django_messages/migrations/0001_initial.py

```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.db import models, migrations
 from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
Is this import needed? Can't we just use on_delete=models.CASCADE below?

Reply:
Agreed